### ISSUE TITLE
Connect itch.io account

### ISSUE THREAD

**Issue by aggsol** [liberapay/liberapay.com#523]
Feature request: Connect to [itch.io](https://itch.io/) account and profile.

**Comment by Changaco** [comment]
We should talk to the itch.io folks about this.

**Comment by aggsol** [comment]
@Changaco You are right,[ they already have a feature request for recurring donations](https://github.com/itchio/itch.io/issues/447) adn there is no need for them to implement again if they could connect with liberapay.
